WPFClient/Pages/Login.xaml

```diff
@@ -52,8 +52,8 @@
                 Click="loginButton_Click"/>
             <Label x:Name="statusLabel"
                    Margin="0,40,0,0" Foreground="#FFF10F0F"
-                   HorizontalContentAlignment="Center" Grid.Row="1" Grid.ColumnSpan="2"
-                   Content="{Binding Source={x:Static local:MainWindow.Status}}"/>
+                   HorizontalContentAlignment="Center" Grid.Row="1" Grid.ColumnSpan="2"/>
+            <!--Content="{Binding Source={x:Static local:Login.Status}}"-->
```

Comment:
Is this commented out because you may want to use it later? or can the line just be deleted?

Reply:
I was trying different things to see if I could get the server status to work.. tbh, I think refactoring to MVVM could make all of our binding much easier